azext_iot/iothub/params.py

```diff
@@ -246,3 +247,133 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot hub messaging-endpoint") as context:
```

Comment:
I think we should go with message-endpoint over messaging-endpoint for the command group. Then for routes we can do message-route.